mlflow/tracking/fluent.py

```diff
@@ -182,10 +207,33 @@ def active_run():
     (parameters, metrics, etc.) through the run returned by ``mlflow.active_run``. In order
     to access such attributes, use the :py:class:`mlflow.tracking.MlflowClient` as follows:
 
-    .. code-block:: py
+    .. code-block:: python
+        :caption: Example
+
+        from pprint import pprint
+        import mlflow
 
         client = mlflow.tracking.MlflowClient()
         data = client.get_run(mlflow.active_run().info.run_id).data
+
+        # Extract params, metric, and tags from data
+        params = {k: v for k, v in data.params.items()}
+        metrics = {k: v for k, v in data.metrics.items()}
```

Comment:
why do we need use dictionary comprehension here?

Reply:
I used to create a dictionary of k-v for each MLflow entities, making it easier to pretty print. We have used a similar code in our [scikit-learn auto logging example](https://mlflow.org/docs/latest/python_api/mlflow.sklearn.html#mlflow.sklearn.autolog).

Reply:
`params` and `metrics` are dict by default:

```python
import mlflow

with mlflow.start_run() as run:
    mlflow.log_params({"a": 0, "b": 1})
    mlflow.log_metrics({"c": 2, "d": 3})

client = mlflow.tracking.MlflowClient()
data = client.get_run(run.info.run_id).data

print(type(data.params))
print(type(data.metrics))
```

output:

```
print(type(data.params))
print(type(data.metrics))
```
```

Reply:
The reason I use dictionary comprehension for **`tags`** is filter out MLflow system tags.

Reply:
Consider it done!